SDWebImage/Core/SDImageCache.h

```diff
@@ -235,6 +235,15 @@ typedef NS_OPTIONS(NSUInteger, SDImageCacheOptions) {
  */
 - (nullable NSData *)diskImageDataForKey:(nullable NSString *)key;
 
+/**
+ * Asynchronously load the image data in disk cache. You can decode the image data to image after loaded.
+ *
+ *  @param key The unique key used to store the wanted image
+ *  @param completionBlock the block to be executed when the check is done.
+ *  @note the completion block will be always executed on the main queue
+ */
+- (void)diskImageDataQueryForKey:(nullable NSString *)key completion:(nullable SDImageCacheQueryDataCompletionBlock)completionBlock;
```

Comment:
Why this API looks a little strange (not `diskImageDataForKey:completion:`), because of the Swift API convension.
This API will convert to Swift as `diskImageData(forKey:completion=nil)`, which allows to ignore the last arg. So it conflict with the exist one `diskImageData(forKey:)`... I don't know how good solution for this, so I add the `Query` as suffix.

Reply:
Keep this API naming currently. May change in the future if we find some better naming convention rule :)